preliz/predictive/ppe.py

```diff
@@ -58,4 +61,4 @@ def ppe(model, target):
     # So we can write it as a PyMC model
     new_priors = backfitting(prior, p_model, var_info2)
 
-    return prior, write_pymc_string(new_priors, var_info2)
+    return prior, new_priors, write_pymc_string(new_priors, var_info2)
```

Comment:
Any reasons why we don't return the `new_priors`?
I think we can return it as well, easier to test/debug the code.

Reply:
what to return is something open to discussion, for end-user I think the most useful result is a string with the new model so they can use it to write a new model, so that's the only thing we should return. For the moment is ok to return many elements, then we can decide what to return by default and maybe option to return more elements

Reply:
Sounds good we keep the new_priors as of now